src/orders/service/JsonOrderFormatterService.ts

```diff
@@ -58,4 +64,24 @@ export default class JsonOrderFormatterService {
     return tax;
   }
 
+  private static formatRefund(refund: Refund): Refund {
```

Comment:
This is probably a bug currently, but we should map the line items for refunds here as well so that the numbers get converted. Should just be able to copy how we are doing it on fulfillments.